docs/howtos/howto.md

```diff
@@ -0,0 +1,45 @@
+# nyaml Installation and Command
+`nyaml` is a Python package that is [published on PyPI](https://pypi.org/project/nyaml/).
+
+## How to Install nyaml
```

Comment:
I can even see this working without mentioning always nyaml, i.e. "How to install"
"How to use"

Reply:
Addressed!